concent-builder/role-tiller.yml

```diff
@@ -0,0 +1,49 @@
+# Tiller permission to all resources in default namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-manager
```

Comment:
This role itself is not related to tiller in anyway. It should not have `tiller` in its name.

It's a generic role that allows doing anything in the `default` namespace. So why not `default-namespace-manager`? Or `default-namespace-admin`?

Reply:
Done.